libgeotiff/libgeotiff.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: @PACKAGE_NAME@
+Description: GeoTIFF file format library
+Version: @VERSION@
+Libs: "-L${libdir}" -lgeotiff
+Cflags: "-I${includedir}"
```

Comment:
This is incomplete without `Libs.private` /  `Requires.private` for static linkage requirements.

Reply:
https://github.com/OSGeo/libgeotiff/blob/7a36257379583447d734fcd8accf5d18701c9621/libgeotiff/libgeotiff.pc.in#L11
Added `Requires.private` only, since both `libtiff` and `proj` already have support to `pkg-config`.